src/Interpreters/Context.cpp

```diff
@@ -3303,14 +3304,14 @@ void Context::clearUncompressedCache() const
         cache->clear();
 }
 
-void Context::setPageCache(size_t bytes_per_chunk, size_t bytes_per_mmap, size_t bytes_total, bool use_madv_free, bool use_huge_pages)
+void Context::setPageCache(size_t default_block_size, size_t default_lookahead_blocks, std::chrono::milliseconds history_window, const String & cache_policy, double size_ratio, size_t min_size_in_bytes, size_t max_size_in_bytes, double free_memory_ratio)
```

Comment:
```suggestion
void Context::setPageCache(
    size_t default_block_size,
    size_t default_lookahead_blocks,
    std::chrono::milliseconds history_window,
    const String & cache_policy,
    double size_ratio,
    size_t min_size_in_bytes,
    size_t max_size_in_bytes,
    double free_memory_ratio)
```

Reply:
Can I at least have a few per line? :) That's so many added lines for little added readablility.

Reply:
I'll allow it